README.md

```diff
@@ -69,6 +72,17 @@ gitlab_ci_runner::runners:
     ensure: absent
 ```
 
+## SLES
+
+There are no gitlab_ci_runner repositories for SLES/zypper available!
+Please set the following data to be able to use this module on SLES:
+
+```yaml
+gitlab_ci_runner::install_method: 'binary' # required for SLES
```

Comment:
please document that the `puppet/archive` module is required for this method.

Reply:
Done